ncm-spma/src/main/pan/components/spma/yumng/config.pan

```diff
@@ -0,0 +1,22 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+unique template components/spma/yumng/config;
+
+# Prefix for packages/groups
+prefix '/software';
+'groups/yumng/names' ?= list();
```

Comment:
This does not match line 29 of ncm-spma/src/main/perl/spma/yumng.pm. If schema.pan sets it to list() by default then this line can be dropped.